packages/lib/modules/lbp/steps/SaleStructureStep.tsx

```diff
@@ -181,14 +181,16 @@ export function SaleStructureStep() {
                 The initial seed amounts and ratio set the starting price, projected market cap and
                 price curve.
               </Text>
-              <Alert status={saleStartsSoon ? 'warning' : 'info'} variant="WideOnDesktop">
-                <AlertIcon as={saleStartsSoon ? AlertTriangle : LightbulbIcon} />
-                <AlertDescription>
-                  {saleStartsSoon && 'This sale is scheduled to start soon. '}
-                  {`The LBP will fail to launch unless you seed the initial liquidity before the
-                  scheduled start time at ${format(parseISO(saleStructureData.startTime), 'h:mmaaa, d MMMM yyyy')}.`}
-                </AlertDescription>
-              </Alert>
+              {saleStart && (
```

Comment:
I think this was already addressed by Matt at https://github.com/balancer/frontend-monorepo/pull/1321

Reply:
ah i thought that was another issue, will update this PR